src/foam/u2/IFrameHTMLView.js

```diff
@@ -39,10 +39,7 @@ foam.CLASS({
         el.contentDocument.documentElement.scrollHeight,
         el.contentDocument.body.firstElementChild.scrollHeight
       );
-      el.style.width = Math.max(
-        el.contentDocument.documentElement.scrollWidth,
-        el.contentDocument.body.firstElementChild.scrollWidth
-      );
+      el.style.width = "100%";
```

Comment:
Feel like we might have situations where we need both use cases, but I can't think of anything breaking off the top of my head so we can merge and then maybe add a property later